TwoPageEBookReader/app/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
+
+    defaultConfig {
+        applicationId "com.example.ebook_reader_sample"
+        minSdkVersion 21
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+}
+
+dependencies {
+
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    implementation 'androidx.core:core-ktx:1.5.0'
+    implementation 'androidx.appcompat:appcompat:1.3.0'
+    implementation 'com.google.android.material:material:1.3.0'
+    implementation "androidx.viewpager2:viewpager2:1.0.0"
+    implementation 'androidx.window:window:1.0.0-alpha08'
+    implementation "androidx.slidingpanelayout:slidingpanelayout:1.2.0-alpha02"
+    implementation 'com.microsoft.design:fluent-system-icons:1.1.133@aar'
```

Comment:
I've seen that in the project repo they say to get the artifact writing the `@aar` part (that to me is a bit weird), but if you search in Maven Central, where they have the artifact published, you can see how you can get the library as any other by doing:
`implementation 'com.microsoft.design:fluent-system-icons:1.1.135'`

Please, see: https://search.maven.org/artifact/com.microsoft.design/fluent-system-icons/1.1.135/aar

Probably is worth to try (I could be wrong and @aar usage is mandatory...)

Reply:
you're right, it works